packages/atlas/src/views/global/YppLandingView/YppRewardSection.styles.ts

```diff
@@ -19,14 +20,33 @@ export const BenefitsCardsButtonsGroup = styled.div`
     display: none;
   }
 
-  ${BenefitsCardButton}:not(:last-child) {
-    margin-right: ${sizes(2)};
+  ${media.sm} {
+    grid-template-columns: repeat(2, 1fr);
+    ${BenefitsCardButton}:not(:nth-child(2n)) {
+      margin-right: ${sizes(2)};
+    }
```

Comment:
I think instead of this (and the md and lg cases too) you could just use a `gap: ${sizes(2)};`

Reply:
I meant the `gap` would be applied to the parent element regardless of the viewport size